src/modules/person/api/person.response.ts

```diff
@@ -0,0 +1,28 @@
+import { AutoMap } from '@automapper/classes';
+
+export interface NameOfPerson {
+    firstName: string;
+    lastName: string;
+    initialsFirstName?: string;
+    initialsLastName?: string;
+    nameSortIndex?: string;
+}
```

Comment:
Should be a separate class with Expose and ApiProperty decorators. `create-person.body.params.ts` is a good example.

Reply:
a person-name.params was found in the api folder. It will be used instead.